### PR TITLE
BZ 1528705 Add information about AWS Service Broker

### PR DESCRIPTION
**Pull request by ghost**
https://bugzilla.redhat.com/show_bug.cgi?id=1528705

### CODE REVIEW

**Comment by tmorriso-rh** [comment]
@bmcelvee Hi Brandi! Can you do the peer review on this since it's related to Developer Experience? Thanks!

File: release_notes/ocp_3_7_release_notes.adoc [hunk 1]
```diff
@@ -1172,6 +1172,17 @@ Cluster Metrics] for more information.
 [[ocp-37-developer-experience]]
 === Developer Experience
 
+[[ocp-37-aws-service-broker]]
+==== AWS Service Broker
+
+Users can seamlessly configure, deploy, and scale AWS services like Amazon RDS,
+Amazon Aurora, Amazon Athena, Amazon Route 53, and AWS Elastic Load Balancing
+directly within the {product-title} console. For installation instructions, see
+link:https://github.com/awslabs/aws-servicebroker-documentation/wiki[AWS Service
```
Comment (bmcelvee) [inline release_notes/ocp_3_7_release_notes.adoc]: The link points to: https://github.com/awslabs/aws-servicebroker/

Do you think it should go directly to the new repo?

**Comment by bmcelvee** [comment]
One question, otherwise LGTM :tulip:

**Comment by tmorriso-rh** [comment]
Thanks @bmcelvee. I updated the link.

**Pull request merged**